Delete stale instances: add never policy to prevent unwanted deletions (#534)
netgroup-polito/CrownLabs: operators/cmd/delete-stale-instances/README.md

# Delete Stale Instances

## About the application

- The application deletes all active instances that are expired.

> - The expiration threshold is retrieved from corresponding template (i.e, the `deleteAfter` field). Therefore who creates the instance is not able to modify (for now) the expiration time, but she inherits it from the template.

Reviewer: ```suggestion
- The expiration threshold is retrieved from the corresponding template (i.e, the `deleteAfter` field). Therefore who creates the instance is not able to modify (for now) the expiration time, since it is inherited from the template.
```
Not sure _she_ can be applicable to she.

Author: IMO, it refers to *who creates the instance*, hence it should be either he or she.
